tools/bep_gant_chart.py

```diff
@@ -0,0 +1,116 @@
+"""Creates a gant chart for the completed BEPs"""
+
+from pathlib import Path
+from rich import print
+import ruamel.yaml as yaml
+import plotly.express as px
+import pandas as pd
+import datetime
+
+root_dir = Path(__file__).parent.parent
+
+completd_beps = root_dir / "_data" / "beps_completed.yml"
+
+BIDS_RELEASES = [
+    "1.0.0",
+    "1.0.1",
+    "1.0.2",
+    "1.1.0",
+    "1.1.1",
+    "1.1.2",
+    "1.2.0",
+    "1.2.1",
+    "1.2.2",
+    "1.3.0",
+    "1.4.0",
+    "1.4.1",
+    "1.5.0",
+    "1.6.0",
+    "1.7.0",
+    "1.8.0",
+]
+
+BIDS_RELEASES_DATES = [
+    "2016-06",
+    "2017-03",
+    "2017-07",
+    "2018-04",
+    "2018-06",
+    "2019-01",
+    "2019-05",
+    "2019-08",
+    "2020-02",
+    "2020-04",
+    "2020-06",
+    "2020-10",
+    "2021-02",
+    "2021-04",
+    "2022-02",
+    "2022-10",
```

Comment:
need to add this to the BIDS spec release protocol that an entry is pushed here?

Reply:
OK won't do it here and not sure this requires automation given how often we release